server/index.js

```diff
@@ -1,6 +1,11 @@
 const browserify = require('browserify-middleware');
 const express = require('express');
 const bodyParser = require('body-parser');
+const socket = require('socket.io-client')('http://localhost');
+
```

Comment:
This should be `require('socket.io')`
The listening portion will happen later down in the file